samples/sample/src/commonTest/kotlin/com/careem/mockingbird/sample/Sample.kt

```diff
@@ -82,5 +82,11 @@ class TestClass {
         val mock = ExternalDepMock()
         assertNotNull(mock)
     }
+
+    @Test
+    fun testGeneratedInternalClass() {
```

Comment:
isnt there a better way to test generated code from kotlin poet? this doesnt really test the change you are applying here

Reply:
https://github.com/careem/mockingbird/issues/95

Reply:
True @carlonzo, because we will get compilation error once the internal interface has a generated mock. Just adding the interface in the list of mocks should be enough to reproduce the visibility bug.